akka-remote/src/main/resources/reference.conf

```diff
@@ -1035,6 +1035,64 @@ akka {
         # it doesn't have to be thread-safe.
         # Refer to `akka.remote.artery.RemoteInstrument` for more information.
         instruments = ${?akka.remote.artery.advanced.instruments} []
+
+      }
+
+      # SSL configuration that is used when transport=tls-tcp.
+      ssl {
+        # Factory of SSLEngine.
+        # Must implement akka.remote.artery.SSLContext and have a public constructor with
+        # optional ActorSystem parameter.
+        ssl-engine-provider = akka.remote.artery.ConfigSSLEngineProvider
+
+        # This is the Java Key Store used by the server connection
+        key-store = "keystore"
+
+        # This password is used for decrypting the key store
+        key-store-password = "changeme"
```

Comment:
in docs for this we should show how to pass in this via an env variable into config I think (so people dont put passwords in config files)

Reply:
👍